docs/data/toolpad/core/components/dashboard-layout/dashboard-layout.md

```diff
@@ -27,3 +27,9 @@ The `branding` prop in the [AppProvider](https://mui.com/toolpad/core/react-app-
 The `navigation` prop in the [AppProvider](https://mui.com/toolpad/core/react-app-provider/) allows for setting any type of navigation structure in the sidebar, such as links, headings, nested collapsible lists and dividers, in any order.
 
 {{"demo": "DashboardLayoutNavigation.js", "height": 640, "iframe": true}}
+
+## Account
```

Comment:
I don't mind the name `UserAccount` if we used it by the way, seems clearer to me.

Reply:
The issue with components starting with `use` is that the docs builder scripts recognise them as hooks. To me `<Account />` seems okay. We can take up modifying the script in case it seems unusable

Reply:
Ok, if that's the main/only reason I guess we can make the part that checks "use" in the scripts be case sensitive? Anyway it's not that important to change this name, only if everyone agrees.